tests/unit/test-cmd/test.run.js

```diff
@@ -161,4 +163,10 @@ describe('run', () => {
     assert.equal(getFakeManifest.called, true);
   });
 
+  it('returns ExtensonRunner', async () => {
+    const cmd = prepareRun();
+    const extensionRunner = await cmd.run();
+    assert.instanceOf(extensionRunner, FakeMultiExtensionRunner);
```

Comment:
was looking for a way to test if the interface was implemented but I'm new to this syntax and could not figure it out.